msteams-platform/graph-api/activity-feed/feed-notifications.md

```diff
@@ -0,0 +1,171 @@
+- The  **Client secret/password** or a public/private key pair (**Certificate**). This is not required for native apps.
+- A **Redirect URI** (or reply URL) for your app to receive responses from Azure AD.
+
+ *See* [Get access on behalf of a user](/graph/auth-v2-user?view=graph-rest-1.0#3-get-a-token) and [Get access without a user](/graph/auth-v2-service).
+
+## Update your Teams app manifest
+
+> [!div class="checklist"]
+> [!div class="checklist"]
+> - Add a [webApplicationInfo](../../resources/schema/manifest-schema.md#webapplicationinfo) key to your app manifest with the following values:  
+&emsp;&emsp; &#8226; **id**  — your Azure AD app id.  
+&emsp;&emsp; &#8226;  **resource** — the resource url for the app.
+>
+> [!div class="checklist"]
+> [!div class="checklist"]
+>
+> - Add an **activities** key to your app manifest with an `activityType array`  value. Each activity type object will have the following properties:  
+&emsp;&emsp; &#8226; **type**  
+&emsp;&emsp; &#8226;  **description**  
+&emsp;&emsp; &#8226;  **templateText**
+
+```json
+"webApplicationInfo": {
+    "id": "73ee2834-38aa-4077-b4c9-e8e1c36f7e40",
+    "resource": "https://AkjStoreApp"
+  },
+  "activities": {
+    "activityTypes": [
+      {
+        "type": "taskCreated",
+        "description": "Task Created Activity",
+        "templateText": "{actor} created task {taskId} for you"
+      },
+      {
+        "type": "teamMention",
+        "description": "Team Mention Activity",
+        "templateText": "{actor} mentioned random team"
+      },
+      {
+        "type": "channelMention",
+        "description": "Channel Mention Activity",
+        "templateText": "{actor} mentioned random channel"
+      },
+      {
+        "type": "userMention",
+        "description": "Personal Mention Activity",
+        "templateText": "{actor} mentioned you dude"
+      },
+      {
+        "type": "calendarForward",
+        "description": "Forwarding a calendar event",
+        "templateText": "{actor} sent you an invite on behalf of {eventOwner}"
+      },
+      {
+        "type": "creatorTaskCreated",
+        "description": "Created Task Created",
+        "templateText": "The Creator created task {taskId} for you"
+      }
+    ]
+  },
+```
+
+## Activity feed notification payload properties
+
+| Property | Type | Required | Description |
+|-----|------|------|-----|
+|**topic**|teamworkActivityTopic | ✔ | represents what is being referenced in the feed item|
+|**activityType** | string | ✔| Represents the type of activity and must be declared in the [Teams App Manifest](../../resources/schema/manifest-schema.md)|
+|**recipient** | teamworkNotificationRecipient |  ✔ | The intended receiver. A recipient must be a Teams user with the ability to post notifications to everyone in a team, channel, and chat.|
```

Comment:
Technically not required if you're using the 'users/{userId}/teamwork/sendActivityNotification' endpoint. In this case the recipient is simply the 'userId' specified in the URI (i.e. the binding parameter).

Reply:
Actually, it should be made clear that if they include the recipient in the payload when using the above URI, we will throw back an error status code.

Reply:
Hi @eddie-lee-msft. It's not clear which (if any) payload properties can be exposed. Can you let me know? Thanks!
![image](https://user-images.githubusercontent.com/50501194/83929917-fdc1a900-a749-11ea-8be6-4d0a4a53bc17.png)

Reply:
All but the one I commented on (teamsAppId and from) can be exposed :)